server/app/views/applicant/ProgramIndexTemplate.html

```diff
@@ -1,11 +1,62 @@
+<!--/* 
```

Comment:
Praise: I like this comment on the top of the template with a list of parameters. I should add it to the other pages